crypto/params.c

```diff
@@ -258,6 +260,15 @@ int OSSL_PARAM_get_int(const OSSL_PARAM *p, int *val)
         return OSSL_PARAM_get_int64(p, (int64_t *)val);
     }
 #endif
+
```

Comment:
This should go before the `#ifndef OPENSSL_SMALL_FOOTPRINT` block.  Both here and everywhere else that has such.

Reply:
No, that would be wrong

Reply:
Small footprint doesn't mean no checks for invalid input